internal/p2p/peermanager_scoring_test.go

```diff
@@ -97,3 +97,176 @@ func TestPeerScoring(t *testing.T) {
 		}
 	})
 }
+
+func makeMockPeerStore(t *testing.T, peers ...peerInfo) *peerStore {
+	t.Helper()
+	s, err := newPeerStore(dbm.NewMemDB())
+	if err != nil {
+		t.Fatal(err)
+	}
+	for idx := range peers {
+		if err := s.Set(peers[idx]); err != nil {
+			t.Fatal(err)
+		}
+	}
+	return s
+}
+
+func TestPeerRanking(t *testing.T) {
+	t.Run("InactiveSecond", func(t *testing.T) {
+		store := makeMockPeerStore(t,
+			peerInfo{
+				ID:       "second",
+				Inactive: true,
+			},
+			peerInfo{
+				ID:       "first",
+				Inactive: false,
+			})
```

Comment:
Minor entirely optional legibility suggestion:
```suggestion
		store := makeMockPeerStore(t,
			peerInfo{ID: "second", Inactive: true},
			peerInfo{ID: "first", Inactive: false},
		)
```